app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Router for all users </h1>
\ No newline at end of file
+<% @users.each do |user| %>
+  <div clas='main-container'>
+    <div class='user-container'>
+      <img src="<%= user.photo %>" class='user-image'>
+    </div>
+    <h2 class='users-info'>
+      <%= link_to user.name, user_path(user.id) %>
+    </h2>
+    <p class='post-num'>Number of posts: <%= user.posts_counter%></p>
```

Comment:
- [Optional] You did no effort to add some stylings to match the given wireframe. I suggest you add some styling since it is part of the learning process though it wasn't a strict requirement. You can see the differences below:

| Wireframe :+1:  | Your design |
| --- | --- |
| ![Screenshot from 2023-06-01 10-27-40](https://github.com/josuerushanika/Blog_App/assets/63087491/93757384-26fc-4eb1-9790-980b3ce7052e) | ![Screenshot from 2023-06-01 10-27-55](https://github.com/josuerushanika/Blog_App/assets/63087491/a991a6b4-7428-477d-afe7-133aa30b24d9) |